arviz/plots/densityplot.py

```diff
@@ -42,6 +43,10 @@ def plot_density(
     data_labels : list[str]
         List with names for the samples in the list of datasets. Useful when
         plotting more than one trace.
+    group: str, optional
+        Which sort of data should be plotted?  Defaults to 'posterior'.  Alternative
```

Comment:
I would change this docstring to a statement rather than a question. I would also move `Default to "posterior" to the end to match other doc strings.

`hpd_markers` and `point_estimate` docstrings in the same function are good examples.

Reply:
I will make the suggested changes and push a revised commit.
But one question: when you say move the default to the end, I think the resulting docstring will be incoherent, because the sentence which will be left in the middle describes alternatives to the default.  It reads a lot better _after_ the specification of the default.

From my perspective as a new user, the notion of "group" is a little mysterious.  I *think* that the group could be any string, but in current practice is limited to 'posterior', 'prior', and 'posterior_predictive'.  Is that correct? Is this documented somewhere?